platform-hub-web/src/app/kubernetes-tokens/kubernetes-tokens-list.component.js

```diff
@@ -0,0 +1,100 @@
+export const KubernetesTokensListComponent = {
+  template: require('./kubernetes-tokens-list.html'),
+  controller: KubernetesTokensListController
+};
+
+function KubernetesTokensListController(roleCheckerService, hubApiService, logger, $stateParams, $mdDialog) {
+  'ngInject';
+
+  const ctrl = this;
+  const userId = $stateParams.userId;
```

Comment:
Likewise, shouldn't be using `$stateParams` here either. (See https://ui-router.github.io/guide/ng1/migrate-to-1_0#stateparams-deprecation for details)